CMakeLists.txt

```diff
@@ -438,8 +437,10 @@ CHECK_TYPE_SIZE("SSIZE_T" EVENT__SIZEOF_SSIZE_T_UPPER LANGUAGE C)
 
 if (EVENT__SIZEOF_SSIZE_T_LOWER)
     set(EVENT__ssize_t "ssize_t")
+    set(EVENT__SIZEOF_SSIZE_T ${EVENT__SIZEOF_SSIZE_T_LOWER})
```

Comment:
Nope, I'm going to leave it as-is, since we have `EVENT__SIZEOF_OFF_T`, sorry for the noise